nilearn/_utils/niimg.py

```diff
@@ -154,7 +156,7 @@ def copy_img(img):
 
     if not isinstance(img, nibabel.spatialimages.SpatialImage):
         raise ValueError("Input value is not an image")
-    return new_img_like(img, _safe_get_data(img).copy(), img.affine.copy(),
+    return new_img_like(img, _safe_get_data(img, copy_data=True), img.affine.copy(),
```

Comment:
Is this the only case where the data was copied ?

Reply:
It was the only call that broke some tests and caught my attention. 
You're right though, there are other calls that make an explicit copy of the data. I updated the code to handle them in c068665.